raven/build.gradle

```diff
@@ -41,9 +41,10 @@ dependencies {
     compile 'org.opencadc:cadc-inventory:[0.1,)'
     compile 'org.opencadc:cadc-inventory-db:[0.1,)'
     compile 'org.opencadc:cadc-storage-permissions:[0.1,)'
+    compile 'org.opencadc:cadc-storage-permissions-client:[0.1,)'
     compile 'org.opencadc:cadc-vos:[1.1.8,)'
-    
-    runtime 'org.opencadc:cadc-access-control:[1.1.1,2.0)'
+    compile 'org.opencadc:cadc-access-control:[1.1.1,2.0)'
```

Comment:
The compile dependency should be cadc-gms and use that API in the code. For now, we do need to include a runtime dependency on cadc-access-control for the implementation.

Same for the minoc/build.gradle as well

Reply:
Updated.